FND/DatesAndTimes/Occurrences.rdf

```diff
@@ -177,24 +184,25 @@ They are modularized this way to minimize the ontological committments that are
 				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>occurrence based date</rdfs:label>
-		<skos:definition>An OccurrenceBaseDate is a CalculatedDate that is defined with respect to the Occurrence of some OccurrenceKind. The &apos;hasDateValue&apos; property of an OccurrenceBasedDate is not set until the Occurrence happens.  The &apos;triggeredBy&apos; property relates an OccurrenceBasedDate to the OccurrenceKind that gives the meaning of the OccurrenceBasedDate.</skos:definition>
+		<rdfs:label>occurrence-based date</rdfs:label>
+		<skos:definition>calculated date that is defined with respect to the occurrence of some occurrence kind</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>The &apos;hasDateValue&apos; property of an OccurrenceBasedDate is not set until the Occurrence happens.  The &apos;triggeredBy&apos; property relates an OccurrenceBasedDate to the OccurrenceKind that gives the meaning of the OccurrenceBasedDate.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-dt-oc;OccurrenceKind">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;Classifier"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-dt-oc;hasDescription"/>
-				<owl:onDataRange rdf:resource="&xsd;string"/>
-				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+				<owl:onProperty rdf:resource="&lcc-cr;classifies"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-oc;Occurrence"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>occurrence kind</rdfs:label>
-		<skos:definition>instances of OccurrenceKind are types of events, each having a description</skos:definition>
-		<skos:editorialNote>In order for other ontologies to accept FinancialDates without committing to the particular notions of &apos;Occurrence&apos; and &apos;OccurrenceKind&apos; that is modeled here, all aspects of Occurrences are captured in this ontolog</skos:editorialNote>
-		<skos:example>Loan origination</skos:example>
-		<skos:example>Trade settlement</skos:example>
+		<skos:definition>classifier that specifies the general nature of an occurrance (event)</skos:definition>
```

Comment:
typo - occurrence has no "a". 
Also I think "the general nature" could be more specific.Examples would be useful.